native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js

```diff
@@ -16,7 +16,13 @@
  * limitations under the License.
  */
 
-import { View, Platform, StyleSheet, ViewPropTypes } from 'react-native';
+import {
+  Platform,
+  StyleSheet,
+  TouchableNativeFeedback,
```

Comment:
If we're going to use our `BpkTouchableNativeFeedback` wrapper, maybe we could expose `TouchableNativeFeedback.SelectableBackground()` through there, removing the need to import both that and  `TouchableNativeFeedback`. What do you think?

Reply:
I don't care that much, but I'm lazy so I lean towards just keeping it like this. cc @matthewdavidson

Reply:
It's strange from a consumer POV to import `BpkTouchableNativeFeedback` as well as `TouchableNativeFeedback`. Can we add a prop?